### PR TITLE
fix(ArtistBiography): use text variant sm

### PR DESCRIPTION
**Pull request by araujobarret**
This PR resolves [DIA-458] 

### Description

Changes the text variant of ArtistBio to `sm` instead of `sm-display`.

PS: no screenshot this time, made from Github VS code.

### PR Checklist

- [ ] I have tested my changes on **iOS** and **Android**.
- [ ] I hid my changes behind a **[feature flag]**, or they don't need one.
- [ ] I have included **screenshots** or **videos**, or I have not changed the UI.
- [ ] I have added **tests**, or my changes don't require any.
- [ ] I added an **[app state migration]**, or my changes do not require one.
- [ ] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [ ] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- fix: change artist bio variant to sm - araujobarret

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[DIA-458]: https://artsyproduct.atlassian.net/browse/DIA-458?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- Cross-platform user-facing changes (fix: change artist bio variant to sm - araujobarret)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 963fcfa0021b070b38fc353c92819f51ebc395d0
</p>

File: src/app/Components/Artist/Biography.tsx [hunk 1]
```diff
@@ -32,7 +32,7 @@ export const Biography: React.FC<BiographyProps> = ({ artist }) => {
         html={`${expanded ? text : truncatedText}${
           text.length > MAX_CHARS && !expanded ? "... " : " "
         }`}
-        variant="sm-display"
+        variant="sm"
```
Comment (anandaroop) [inline src/app/Components/Artist/Biography.tsx:35]: If I'm following correctly, that will change the line-height from 20 to 26 👍🏽 

https://github.com/artsy/palette/blob/60f91aef456c0ab2aafad1ad71e52053171aa9fb/packages/palette-tokens/src/typography/v3.ts#L62-L69

**Review by anandaroop — APPROVED** [review]
🙏🏽

File: src/app/Components/Artist/Biography.tsx (hunk 1 above)
Comment (araujobarret) [reply src/app/Components/Artist/Biography.tsx:35]: yep, we use `sm-display` for a more condensed text idea from what I see in our designs, the Bio case is a long text, should be nicer to read than a pill prescription paper 😂

**Pull request merged**